data/assets/scene/solarsystem/missions/voyagerpioneer/voyager1_2__pioneer10_11.asset

```diff
@@ -13,8 +13,6 @@ local pioneer_horizons = asset.resource({
 })
 
 
-local SampleInterval = 24 * 60 * 60
```

Comment:
Why not store it? Now we need to repeat the calculation 4 times and maintain all of them. Or as I suggested earlier, we could just add a function of number of seconds in a day and use that instead.